Sources/StreamChatUI/ImageGallery/VideoPlaybackControlView.swift

```diff
@@ -0,0 +1,279 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import AVKit
+import StreamChat
+import UIKit
+
+/// A view that shows playback controls and timeline for the given player.
+public typealias VideoPlaybackControlView = _VideoPlaybackControlView<NoExtraData>
+
+/// A view that shows playback controls and timeline for the given player.
+open class _VideoPlaybackControlView<ExtraData: ExtraDataTypes>: _View, ThemeProvider {
+    /// The type describing the content of the view.
+    public struct Content {
+        /// The type describing the current video state.
+        public enum VideoState {
+            case playing
+            case paused
+            case loading
+        }
```

Comment:
Same here, a possible candidate to use the struct with static props pattern, since this is likely to expand and be customizable. Example, a new state the customer might be interested is "seeking"